ShopAppTests/Data/API/Alamofire/BaseAPISpec.swift

```diff
@@ -68,12 +68,14 @@ class BaseAPISpec: QuickSpec {
                     }
                 }
             }
-            
+
```

Comment:
We also should test logic that API client checks cache before sending request

Reply:
See previous comment.